assets/src/scripts/outputs-viewer/App.jsx

```diff
@@ -137,6 +137,8 @@ function App({
               {isReviewEdit ? <FileReview selectedFile={selectedFile} /> : null}
               <Card>
                 <Metadata
+                  context={selectedFile.metadata.context}
```

Comment:
This should be just `metadata=metadata`

The jsx component should iterate through the key/value pairs.

To match the current document, we'd expect a dict like:

````
{
    "Context": "user input context",
    "Controls": "user input controls",
}
```